### DISCUSSION TITLE
Set CSV delimiter per feed, not only globally

### DISCUSSION THREAD

**Discussion by piotrpog** [engram-design/FeedMe#1026] [Ideas]
I have more than one feed set up, which use CSV. 
Some use comma as delimiter, some use semicolon. It would be useful if we could set this delimiter per feed, not only globally using plugin options.

**Comment by gixxy22** [comment]
You can do this by creating a feed-me.php file in /config directory and adding:

```
<?php

return [
    //global override settings
    '*' => [
        'csvColumnDelimiter' => ',',
        'feedOptions' => [
             //per feed override settings (feed id)
            '1' => [
                'csvColumnDelimiter' => ';',
            ]
        ],
    ]
];
```

This would make all feeds default to use comma as a delimiter and a semi-colon for feed number 1.
